recipebox/recipebox/forms.py

```diff
@@ -0,0 +1,21 @@
+from django import forms
+# from django.core.exceptions import ValidationError
+# from django.utils.translation import ugettext_lazy as _
+from recipebox.models import Author, User
+
+
+class RecipeForm(forms.Form):
+    author = forms.ChoiceField(
+        choices=[(a.id, a.user.username) for a in Author.objects.all()])
```

Comment:
This will only work until another Author object is created; the list is cached when the form is called for the first time and never referenced again.